src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/ApprenticeshipDetailsCsvModel.cs

```diff
@@ -17,6 +17,9 @@ public class ApprenticeshipDetailsCsvModel
         [Name("Apprenticeship training course")]
         public string CourseName { get ; private set ; }
 
+        [Name("Planned start date")]
+        public string PlannedStartDate { get; private set; }
```

Comment:
I'm guessing this is now wanted to be shown for employers? Is this only for download though as there's no cshtml change for it.